src/three-components/VisibleSTLModel.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLayerVisibility } from "../contexts/LayerVisibilityContext"
+import { STLModel } from "./STLModel"
+
+interface VisibleSTLModelProps {
+  stlData: ArrayBuffer
+  color: any
+  opacity?: number
+  index: number
+  totalModels: number
+}
+
+export function VisibleSTLModel({
+  stlData,
+  color,
+  opacity = 1,
+  index,
+  totalModels,
```

Comment:
No way. Really bad props. Make the prop meaningful, eg isSilkscreen, isCopper or just visibilityType